config/hoodi.names.yml

```
@@ -45,4 +45,4 @@
 985000-1059999: "attestant"
 
 "withdrawal:0x4473dCDDbf77679A643BdB654dbd86D67F8d32f2": "Lido"
-
+"withdrawal:0x346D827a75F98F0A7a324Ff80b7C3F90252E8baC": "blockshard"
```

```suggestion
"withdrawal:0x346D827a75F98F0A7a324Ff80b7C3F90252E8baC": "Blockshard"
```

small update as all post-genesis validator names start with capital letter :)

---

@mbonenberger can you click accept on the above? :D
I can't do commits to your branch to apply the fix.